### DISCUSSION TITLE
Snapshot of the tweet.

### DISCUSSION THREAD

**Discussion by simanchal-45** [robertodoering/harpy#673] [Ideas]
Add a button to take the snapshot of a perticular tweet.
Just like this👇
![20220527_084433_0000](https://user-images.githubusercontent.com/71957533/170622266-e592fafd-685c-4c0e-a6dd-e50b0c1dc62d.png)

**Comment by robertodoering** [comment]
Nice idea, thanks for the suggestion :)
